crates/miden-lib/src/account/components/mod.rs

```diff
@@ -128,6 +130,7 @@ impl WellKnownComponent {
                     .push(AccountComponentInterface::AuthRpoFalcon512(storage_offset)),
                 Self::RpoFalcon512Acl => component_interface_vec
                     .push(AccountComponentInterface::AuthRpoFalcon512Acl(storage_offset)),
+                Self::NoAuth => component_interface_vec.push(AccountComponentInterface::AuthNoAuth),
```

Comment:
```suggestion
                Self::NoAuth => component_interface_vec.push(AccountComponentInterface::AuthNone),
```

Nit: Not from this PR, afaict, but I feel like this should be called `AuthNone`. The current wording is a bit odd. An alternative could be `AuthIncrNonce`, but maybe this was already decided on in the other PR, so feel free to ignore.

Reply:
Renamed to `AuthNone`